<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PEFE – PDF Extractor for Excel Files</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
      QWidget { background-color: #FFFFFF; }
      QLabel#titleLabel { color: #333333; }
      QLabel#subTitleLabel, QLabel#instructionLabel { color: #777777; }
      QLabel#statusLabel { color: #186F9A; font-size: 12pt; font-weight: bold; }
    </string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0,0,0,0,0,0">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PEFE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="subTitleLabel">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PDF Extractor for Excel Files</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="instructionLabel">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Select a folder containing Excel files to extract PDFs</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacerTop">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="folderButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
         QPushButton {
           background-color: qlineargradient(
             spread:pad, x1:0,y1:0, x2:1,y2:0,
             stop:0 #3A8FB7, stop:1 #186F9A
           );
           color: white;
           border-radius: 8px;
           padding-left: 20px;
           padding-right: 20px;
         }
         QPushButton:hover { background-color: #2E7EA5; }
         QPushButton:pressed { background-color: #155A80; }
       </string>
      </property>
      <property name="text">
       <string>Select Folder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Ready</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="danceLabel">
      <property name="minimumSize">
       <size>
        <width>656</width>
        <height>368</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>656</width>
        <height>368</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <!-- 초기에는 숨김 -->
      <property name="visible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacerMiddle">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="notesGroup">
      <property name="title">
       <string>참고 사항</string>
      </property>
      <layout class="QVBoxLayout" name="notesLayout">
       <item>
        <widget class="QTextBrowser" name="notesText">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacerBottom">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mergeLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="mergeCheckBox">
        <property name="text">
         <string>Merge</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
